docs/script.js

```diff
@@ -0,0 +1,93 @@
+const navMenuButton = document.getElementById("coffee-menu");
+const menuButton = document.getElementById("button-icon-burger");
+const navModal = document.querySelector(".nav-modal");
+const sliderContainer = document.getElementById("sliderContainer");
```

Comment:
choose one naming convention

Reply:
Are we talking about the ( - ) sign in the middle ?